src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -73,8 +78,13 @@ chip::NodeId kRemoteDeviceId       = chip::kTestDeviceNodeId;
 
 extern "C" {
 CHIP_ERROR pychip_DeviceController_NewDeviceController(chip::Controller::DeviceCommissioner ** outDevCtrl,
+                                                       chip::Controller::DeviceAddressUpdater ** outAddressUpdater,
                                                        chip::NodeId localDeviceId);
-CHIP_ERROR pychip_DeviceController_DeleteDeviceController(chip::Controller::DeviceCommissioner * devCtrl);
+CHIP_ERROR pychip_DeviceController_DeleteDeviceController(chip::Controller::DeviceCommissioner * devCtrl,
+                                                          chip::Controller::DeviceAddressUpdater * addressUpdater);
```

Comment:
why does the delete need an address updater?
API seems odd - deleting an object generally only takes that object as an argument

Reply:
Right, I was a bit lazy and adding the python bindings is a bit tedious, but I'll fix it :)